Matrix_Factorial.yml

```diff
@@ -0,0 +1,6 @@
+Name: Matrix_Factorial 
+language: python
+Prompt: Matrix_Factorial.txt
+Label: Matrix_Factorial.py
+context: numpy
+Test: Matrix_Factorial.py
```

Comment:
These are all case sensitive. Better to just make everything lowercase to avoid any issue.